.github/workflows/_integration_test.yml

```diff
@@ -87,3 +87,15 @@ jobs:
         working-directory: libs/cli/js-examples
         run: |
           langgraph build -t langgraph-test-e
+
+      - name: Build JS monorepo service
+        if: steps.changed-files.outputs.all
+        working-directory: libs/cli/js-monorepo-example
+        run: |
+          langgraph build -t langgraph-test-f -c apps/agent/langgraph.json --build-command "yarn run turbo build" --install-command "yarn install"
```

Comment:
i think we want to add a run_langgraph_cli_test.py call and maybe update that one to actually create a run